src/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hello World</title>
+  </head>
+  <body>
+    <p>Hello, world!</p>
+  </body>
+</html>
```

Comment:
always add new line at the end of the file
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
ok, thks!